packages/testsuite/apps/basic/.wundergraph/operations/incremental/data.ts

```
@@ -0,0 +1,42 @@
+import { createOperation } from '../../generated/wundergraph.factory';
+
+export default createOperation.subscription({
+	handler: async function* ({ operations }) {
+		const fastPromise = operations.query({
+			operationName: 'incremental/delay',
+			input: {
+				seconds: 0,
+			},
+		});
+		const slowPromise = operations.query({
+			operationName: 'incremental/delay',
+			input: {
+				seconds: 1,
+			},
+		});
+		const fast = await fastPromise;
+		if (fast.errors) {
```

What's that?

---

@thisisnithin It think you can delete that `if (fast.errors) {` entirely, let it blow up if the operation fails.

---

Simplified test.